src/keywords.cpp

```diff
@@ -287,7 +288,10 @@ static int kw_compare(const void *p1, const void *p2)
    return(strcmp(t1->tag, t2->tag));
 }
 
-
+/**
+ * This function verifies if all language keywords as 'if', 'for', 'break', etc, are 
+ * alphabetycally sorted on the main table.
```

Comment:
It is a documentation. It could seen obvious to you but not to everybody else. The misspelling is fixed.